src/lib/forms/components/LocationDatePicker.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { DatePicker } from '@components/DatePicker';
+import { locationApi } from '@api/locations';
+import { fromISO, toISODate } from '@api/date';
+import _isEmpty from 'lodash/isEmpty';
+
+export class LocationDatePicker extends Component {
```

Comment:
same here, I would move this to the "closure" module...

Reply:
Hmm, this component basically disable days in the calendar depending on the closures of a location. Even it is related to the closures module, it's not used there and it's used in the `LoanRequestForm`, in the `BorrowingRequestPatronLoan` and in the `ESSelectorLoanRequest`. IMHO this one is a little bit more generic and re-usable. SInce it's not only used in the forms we could move it from the `src/lib/forms/components/LocationDatePicker.js` to `src/lib/components/LocationDatePicker.js`. WDYT?